src/azure-openapi-validator/tests/utilities/tests-helper.ts

```diff
@@ -0,0 +1,21 @@
+import { Message } from '../../../jsonrpc/types';
+import { suite, test, slow, timeout, skip, only } from "mocha-typescript";
+import { AutoRestPluginHost } from "../../../jsonrpc/plugin-host";
+import { run } from "../../../azure-openapi-validator";
+
+const fs = require('fs');
+
+export async function CollectTestMessagesFromValidator(openapiDefinitionObject: any): Promise<Message[]> {
+  let messages: Message[] = [];
+  let file = 'fake-test-file';
```

Comment:
Yes, the ```run``` function expects a file path to add to the ```Message``` object it generates, its a simple string, passing some meaningful string made sense here.